SchedulearnBackend/SchedulearnBackend/Controllers/UserController.cs

```diff
@@ -19,11 +23,24 @@ public UserController(UserService userService)
             _userService = userService;
         }
 
+        // POST: api/User/authenticate
+        [AllowAnonymous]
+        [HttpPost("authenticate")]
+        public async Task<ActionResult<string>> Authenticate(AuthenticateModel model)
+        {
+            var token = await _userService.Authenticate(model.Email, model.Password);
+            if (token == null)
```

Comment:
Can this ever be null? I recommend handling exception throwing in service, so its easily testable.